src/ui/registration/containers/entropy.tsx

```diff
@@ -26,7 +26,8 @@ interface State {
 }
 
 // we are gonna collect some from the user and the rest from the OS
-const ENOUGH_ENTROPY_PROGRESS = 0.6
+const ENOUGH_ENTROPY_PROGRESS = 0.3
+const POST_COLLECTION_WAIT_TIME = 1000
```

Comment:
I feel that this wait time seems a bit unnatural - I'm not sure the best course of action though. Instantaneous is also a bit jumpy.

Reply:
Ideally the navigation would be a bit smoother. I don't think it's a matter of the timeout so much, and rather the animation used during navigation. Not entirely sure how to approach this / what the best course of action is.

Reply:
We disabled the animations/transitions between the starting screens - this is probably a casualty of that. I would suggest lowering the timeout for now and then iterating on solutions to make it smoother in the future.